src/main/java/duke/command/Duke.java

```diff
@@ -0,0 +1,176 @@
+package duke.command;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Duke {
+
+
+    public static void main(String[] args) throws IOException {
+
+        List listItem = new List();
```

Comment:
Consider following the naming for list variables and the proper way to declare a list from coding standards